config/config.toml.example

```diff
@@ -226,6 +226,9 @@ stmt-count-limit = 5000
 # Set keep alive option for tcp connection.
 tcp-keep-alive = true
 
+# Set nodelay option for tcp connection.
+tcp-no-delay = false
```

Comment:
This looks inconsistent with the user documentation: https://docs.pingcap.com/tidb/dev/tidb-configuration-file#tcp-no-delay
Can you confirm which default value is correct?